src/main/java/org/spongepowered/common/util/Constants.java

```diff
@@ -1358,5 +1364,25 @@ public static Axis convertAxisToSponge(final EnumFacing.Axis axis) {
                     return Axis.X;
             }
         }
+
```

Comment:
Pretty much all the mappings for the specific block types can exist in here, or as a type provider. That's what I mean by reducing the amount of static usage in the mixins themselves.